ArduCopter/events.cpp

```diff
@@ -100,7 +100,7 @@ void Copter::handle_battery_failsafe(const char *type_str, const int8_t action)
 
     } else if (flightmode->is_landing() && failsafe_option(FailsafeOption::CONTINUE_IF_LANDING) && desired_action != Failsafe_Action_None) {
         // Allow landing to continue when FS_OPTIONS is set to continue when landing
-        desired_action = Failsafe_Action_Land;
```

Comment:
I wonder if it would somehow be easier (or less noticeable a problem) if we implemented the do-land-start so that it took the shortest path to the path described by the commands after the do-land-start (instead of always starting on the command immediately after the do-land-start).

Reply:
@rmackay9 I have branch, I called it mission re-join, does not replace DO_LAND_START, you can use both. That is the follow up to auto rtl mode. We keep finding issues with the mode so I have been holding off.

Maybe it has been in long enough now that it is safe to rebase that and open a PR.